### ISSUE TITLE
PublicApiAnalyzer RS0016 and RS0017 for forwarded types

### ISSUE THREAD

**Issue by RussKie** [dotnet/roslyn-analyzers#3229]
#### Analyzer package

[Microsoft.CodeAnalysis.PublicApiAnalyzer](https://www.nuget.org/packages/Microsoft.CodeAnalysis.PublicApiAnalyzer)

#### Package Version

v2.9.8 (Latest)

#### Diagnostic ID

RS0016, RS0017

#### Repro steps

1. Clone https://github.com/RussKie/winforms.git @ 620263b81406454591ce731c6879ae09d19a3bf1
2. Open in VS
3. Compile in Release mode
4. Check the Output tab

#### Expected behavior

Quick actions are available.

#### Actual behavior

The analyser is complaining (RS0016) that forwarded types aren't declared in the public API list. However attempts to add these API to the list triggers RS0017.

```
4>Properties\TypeForwards.cs(7,12,7,75): warning RS0016: Symbol 'ScreenOrientation' is not part of the declared API.
4>Properties\TypeForwards.cs(7,12,7,75): warning RS0016: Symbol 'Angle0' is not part of the declared API.
4>Properties\TypeForwards.cs(7,12,7,75): warning RS0016: Symbol 'Angle90' is not part of the declared API.
4>Properties\TypeForwards.cs(7,12,7,75): warning RS0016: Symbol 'Angle180' is not part of the declared API.
4>Properties\TypeForwards.cs(7,12,7,75): warning RS0016: Symbol 'Angle270' is not part of the declared API.
4>PublicAPI.Unshipped.txt(10,1,10,39): warning RS0017: Symbol 'System.Windows.Forms.ScreenOrientation' is part of the declared API, but is either not public or could not be found
4>PublicAPI.Unshipped.txt(11,1,11,92): warning RS0017: Symbol 'System.Windows.Forms.ScreenOrientation.Angle0 = 0 -> System.Windows.Forms.ScreenOrientation' is part of the declared API, but is either not public or could not be found
4>PublicAPI.Unshipped.txt(12,1,12,94): warning RS0017: Symbol 'System.Windows.Forms.ScreenOrientation.Angle180 = 2 -> System.Windows.Forms.ScreenOrientation' is part of the declared API, but is either not public or could not be found
4>PublicAPI.Unshipped.txt(13,1,13,94): warning RS0017: Symbol 'System.Windows.Forms.ScreenOrientation.Angle270 = 3 -> System.Windows.Forms.ScreenOrientation' is part of the declared API, but is either not public or could not be found
4>PublicAPI.Unshipped.txt(14,1,14,93): warning RS0017: Symbol 'System.Windows.Forms.ScreenOrientation.Angle90 = 1 -> System.Windows.Forms.ScreenOrientation' is part of the declared API, but is either not public or could not be found
```

**Comment by rponnuru0518** [comment]
+1

**Comment by mavasani** [comment]
Duplicate of https://github.com/dotnet/roslyn-analyzers/issues/3228
